### DISCUSSION TITLE
What do I do with "Long Text"?

### DISCUSSION THREAD

**Discussion by Sagestum** [Budibase/budibase#10999] [Help] [answered]
Hi there, 

I have a long text table in my database and I want to display the content. So I have created a Data Provider, added a Paragraph and selected the binding like this: 
{{ Ticket Comments.Rows.0.TEXT }}

But it does not show the TEXT binding as text, it just prints out "Langtext" (which is German for long text). 

![image](https://github.com/Budibase/budibase/assets/11883823/a9f7f974-d32a-4400-b566-c673b8210c9e)

Is there a way to, well, decript it to text?

I tried it in the SQL query with: SUBSTR(TO_CHAR(td.DESCRIPTION), 1, 4000) AS Text
But it ends with a problem: ORA-22835: Buffer too small for CLOB to CHAR or BLOB to RAW conversion (actual: 7611, maximum: 4000)

thank you 
best
Chris

**Comment by ConorWebb96** [comment]
Hey @Sagestum,

Would you be able to take a few more screenshots of your table structure as well as your data provider settings and your paragraph settings.

If possible could you also screenshot the query you are using?

Lastly what data source are you using? Is it MySQL?

Thanks ahead of time for supplying this information!

**Comment by Sagestum** [comment, marked as answer]
I have now found a solution, I convert it in the Oracle Database with: 

DBMS_LOB.SUBSTR(td.DESCRIPTION, 3900) AS DESCRIPTION,

and works
